sdk/eventhub/event-hubs/README.md

```diff
@@ -371,21 +406,21 @@ export DEBUG=azure*
 export DEBUG=azure*,rhea*
 ```
 
-- If you are **not interested in viewing the event transformation** (which consumes lot of console/disk space) then you can set the `DEBUG` environment variable as follows:
+- If you are **not interested in viewing the raw event data** (which consumes a large amount of console/disk space) then you can set the `DEBUG` environment variable as follows:
 
 ```bash
-export DEBUG=azure*,rhea*,-rhea:raw,-rhea:message,-azure:amqp-common:datatransformer
+export DEBUG=azure*,rhea*,-rhea:raw,-rhea:message
 ```
 
-- If you are interested only in **errors**, then you can set the `DEBUG` environment variable as follows:
+- If you are interested only in **errors** and SDK **warnings**, then you can set the `DEBUG` environment variable as follows:
 
 ```bash
-export DEBUG=azure:event-hubs:error,azure-amqp-common:error,rhea-promise:error,rhea:events,rhea:frames,rhea:io,rhea:flow
+export DEBUG=azure:(event-hubs|core-amqp):(error|warning),rhea-promise:error,rhea:events,rhea:frames,rhea:io,rhea:flow
```

Comment:
Can we not skip the event hubs and core amqp reference here and have
`azure:*:(error|warning)`?

Reply:
You're right, that would work.